SCF.py

```diff
@@ -31,6 +32,27 @@ def calc_nuclear_repulsion_energy(mol_):
     Step 2. Loop over atoms and calculate Enuc from formulat in Readme
     """
 
+    # step1: distance between atoms
+    rows, cols = coords.shape
+    distance = []
+    for i in range(rows):

```

Comment:
Should only use the numpy arrays to do this, shouldn't have needed to use a list. Technically correct but clunky.